tests/modifiers.edge-split-modifier.test.three-js-test.html

```diff
@@ -0,0 +1,724 @@
+        }
+
+        @media (pointer:coarse) {
+            .lil-gui.allow-touch-styles {
+                --widget-height: 28px;
+                --padding: 6px;
+                --spacing: 6px;
+                --font-size: 13px;
+                --input-font-size: 16px;
+                --folder-indent: 10px;
+                --scrollbar-width: 7px;
+                --slider-input-min-width: 50px;
+                --color-input-min-width: 65px
+            }
+        }
+
+        @media (hover:hover) {
+            .lil-gui .controller.color .display:hover:before {
+                border: 1px solid #fff9;
+                border-radius: var(--widget-border-radius);
+                bottom: 0;
+                content: " ";
+                display: block;
+                left: 0;
+                position: absolute;
+                right: 0;
+                top: 0
+            }
+
+            .lil-gui .controller.option .display.focus {
+                background: var(--focus-color)
+            }
+
+            .lil-gui .controller.option .widget:hover .display {
+                background: var(--hover-color)
+            }
+
+            .lil-gui .controller.number .slider:hover {
+                background-color: var(--hover-color)
+            }
+
+            body:not(.lil-gui-dragging) .lil-gui .title:hover {
+                background: var(--title-background-color);
+                opacity: .85
+            }
+
+            .lil-gui .title:focus {
+                text-decoration: underline var(--focus-color)
+            }
+
+            .lil-gui input:hover {
+                background: var(--hover-color)
+            }
+
+            .lil-gui input:active {
+                background: var(--focus-color)
+            }
+
+            .lil-gui input[type=checkbox]:focus {
+                box-shadow: inset 0 0 0 1px var(--focus-color)
+            }
+
+            .lil-gui button:hover {
+                background: var(--hover-color);
+                border-color: var(--hover-color)
+            }
+
+            .lil-gui button:focus {
+                border-color: var(--focus-color)
+            }
+        }
+    </style>
+</head>
+
+<body class="">
+    <script type="importmap">
+        {
+            "imports": {
+                "three": "../node_modules/three/build/three.module.js",
+                "@dioxygen-software/three-js-extra-public": "../dist/three-js-extra-public.module.js",
+                "three/examples/jsm/utils/BufferGeometryUtils.js": "../node_modules/three/examples/jsm/utils/BufferGeometryUtils.js"
```

Comment:
Load the three example directly from node_modules and remove the files in js/ folder